app/models/piece.rb

```diff
@@ -1,3 +1,20 @@
 class Piece < ApplicationRecord
-  belongs_to :user
-	belongs_to :game
+  
+belongs_to :game
+
+def color
+    return 'white' if is_white == true
+    'black'
+  end
+
+  def render
+    "#{color}-#{type.downcase}.png"
+  end
+end 
+
+PAWN = "Pawn".freeze
```

Comment:
Once you do call `Pawn.create` from the previous comment - the constants will become obsolete...

Reply:
I will try that and get rid of the Pawn.freeze to test out.